diff --git a/node_modules/@react-navigation/elements/src/Screen.tsx b/node_modules/@react-navigation/elements/src/Screen.tsx
index 6ea2597..be69edc 100644
--- a/node_modules/@react-navigation/elements/src/Screen.tsx
+++ b/node_modules/@react-navigation/elements/src/Screen.tsx
@@ -16,6 +16,7 @@ import Background from './Background';
 import getDefaultHeaderHeight from './Header/getDefaultHeaderHeight';
 import HeaderHeightContext from './Header/HeaderHeightContext';
 import HeaderShownContext from './Header/HeaderShownContext';
+import Styles from "../../../../consts/Styles";
 
 type Props = {
   focused: boolean;
@@ -99,6 +100,11 @@ const styles = StyleSheet.create({
   // This is necessary to avoid applying 'column-reverse' to screen content
   content: {
     flex: 1,
+    marginTop: -20,
+    zIndex: 10,
+    backgroundColor: 'white',
+    borderTopRightRadius: Styles.contentRadius,
+    borderTopLeftRadius: Styles.contentRadius,
   },
   absolute: {
     position: 'absolute',
